### ISSUE TITLE
UXUI: Request Type Color

### ISSUE THREAD

**Issue by danarchen** [hackforla/311-data#616]
## Description
Current request type colors does not meet ADA standards. Must choose new colors that better meet requirements..

## Action Items

- [x] Research

- [x] Provide new colors to frontend

- [x] Add to key components 

## Resources

- https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1%3A5

**Comment by danarchen** [comment]
New color choices for request type is available on Figma.

Please access through the following link:
https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=1%3A5

**Comment by johnr54321** [comment]
allows for #683
